debian/entry.sh

```diff
@@ -90,6 +90,8 @@ function init_non_systemd()
 		"$CMD" "$@" &
 		pid=$!
 		wait "$pid"
+		exit_code=$?
+		fg > /dev/null || exit "$exit_code"
```

Comment:
`fg` was not deleted in the commit referenced in your commit message. Can you open an issue in this repo that clearly states the problem you're trying to fix here, including a minimal testcase? I don't understand what's going on with the exit codes and what we're trying to fix